tests/plugins/test_check_and_set_platforms.py

```diff
@@ -129,3 +134,48 @@ def test_check_and_set_platforms(tmpdir, platforms, platform_exclude, platform_o
         assert plugin_result[PLUGIN_CHECK_AND_SET_PLATFORMS_KEY] == set(result)
     else:
         assert plugin_result[PLUGIN_CHECK_AND_SET_PLATFORMS_KEY] is None
+
+
+@pytest.mark.parametrize(('labels', 'platforms', 'orchestrator_platforms', 'result'), [
+    ({}, None, None, None),
+    ({}, 'x86_64 arm64', ['spam', 'bacon'], ['arm64', 'x86_64']),
+    ({'isolated': True}, 'spam bacon', ['x86_64', 'arm64'], ['arm64', 'x86_64']),
+    ({'isolated': True}, 'x86_64 arm64', None, ['arm64', 'x86_64']),
```

Comment:
I don't think this is valid use case, as orchestrator platforms will be always set with koji-c